.gitignore

```diff
@@ -10,4 +10,10 @@ export/*
 # Development folder
 dev/
 
+# IDE
+.vscode/
```

Comment:
If we remove .vscode from git, we should delete the current .vscode folder.

Reply:
Actually, I didn't notice that .vscode directory was already part of the repository but thought I introduced it through my VSCode development. However, if the pre-settings are not really necessary, I'd drop that folder but keep the ignore line. 

As you wish.

Reply:
I mean. I like the vscode file in their. It reduces my setup time on different machines. But it's not actually a part of CoinTaxman. I though when you remove this, it might be a good idea in general to keep the repo clean

Reply:
I removed the .vscode Folder (31f172419de955e707fbf340f94b3d000e3c019d).